source/frontend/src/components/common/form/NotesModal.test.tsx

```diff
@@ -59,31 +60,51 @@ describe('NotesModal component', () => {
     const {
       component: { getByTitle, getByText, findByTestId },
     } = await setup({});
-    userEvent.click(getByTitle('notes'));
+    act(() => {
+      userEvent.click(getByTitle('notes'));
+    });
 
     const noteField = await findByTestId('note-field');
-    userEvent.type(noteField, 'test note');
+    act(() => {
+      userEvent.type(noteField, 'test note');
+    });
     const saveButton = getByText('Save');
-    userEvent.click(saveButton);
+    act(() => {
+      userEvent.click(saveButton);
+    });
 
-    userEvent.click(getByTitle('notes'));
-    expect(noteField).toHaveValue('test note');
+    act(() => {
+      userEvent.click(getByTitle('notes'));
+    });
+    await act(async () => {
+      expect(noteField).toHaveValue('test note');
+    });
```

Comment:
Not sure we need to **act** here. Did you try without it? Do you still get warnings?

Reply:
I tried without it and I still get warnings. But as soon as I wrap it in an act(), the warnings go away.

Reply:
would you mind changing the act on 76 to be an await act(async and then removing the act around the expect?

Reply:
That worked so change has been made.